benchmark/src/main/java/com/linecorp/decaton/benchmark/Main.java

```diff
@@ -95,6 +96,9 @@ public final class Main implements Callable<Integer> {
             defaultValue = "text")
     private String resultFormat;
 
+    @Option(names = "--file", description = "Result file to write output to. If not specified, output to stdout")
```

Comment:
ogh.. thanks again for another follow-up. Just a nit, would it be bertter to name this param `--output` following the convention of most command-line utilities?